k8s/base/case-import-cronjob.yaml

```diff
@@ -17,7 +17,13 @@ spec:
                   memory: "384Mi"
               image: docker.io/gridsuite/case-import-job:latest
               volumeMounts:
-                - name: config-case-import-job-volume
+                - mountPath: /config
+                  name: case-import-job-configmap-volume
+                - mountPath: /config/common
+                  name: case-import-job-configmap-common-volume
+                - mountPath: /config/specific
+                  name: case-import-job-configmap-specific-volume
+                - name: config-case-import-job-itools-volume
```

Comment:
rename `config-case-import-job-itools-volume` to `case-import-job-configmap-itools-volume` to be homogenous ?